### ISSUE TITLE
do you have in book xgboost code from scratch as per your post xgboost library in less than 200 lines

### ISSUE THREAD

**Issue by Sandy4321** [PacktPublishing/Hands-On-Gradient-Boosting-with-XGBoost-and-Scikit-learn#3]
do you have in book xgboost code from scratch as per your post xgboost library in less than 200 lines
https://towardsdatascience.com/diy-xgboost-library-in-less-than-200-lines-of-python-69b6bf25e7d9

if not where to find this code in py file pls
